src/main/java/uk/gov/companieshouse/siccode/api/search/CombinedSicActivitiesApiModel.java

```diff
@@ -0,0 +1,130 @@
+    }
+
+    public CombinedSicActivitiesApiModel(String id, String sicCode, String activityDescription,
+             String sicDescription, boolean companiesHouseactivity) {
+        this.id = id;
+        this.sicCode = sicCode;
+        this.activityDescription = activityDescription;
+        this.sicDescription = sicDescription;
+        this.companiesHouseactivity = companiesHouseactivity;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getSicCode() {
+        return sicCode;
+    }
+
+    public void setSicCode(String sicCode) {
+        this.sicCode = sicCode;
+    }
+
+    public String getActivityDescription() {
+        return activityDescription;
+    }
+
+    public void setActivityDescription(String activityDescription) {
+        this.activityDescription = activityDescription;
+    }
+
+    public String getSicDescription() {
+        return sicDescription;
+    }
+
+    public void setSicDescription(String sicDescription) {
+        this.sicDescription = sicDescription;
+    }
+
+    public boolean isCompaniesHouseactivity() {
+        return companiesHouseactivity;
+    }
+
+    public void setCompaniesHouseactivity(boolean companiesHouseactivity) {
+        this.companiesHouseactivity = companiesHouseactivity;
+    }
+
+    // Override both hashCode and equals for testing with Hamcrest matchers (hence want all data attributes)
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((activityDescription == null) ? 0 : activityDescription.hashCode());
+        result = prime * result + ((id == null) ? 0 : id.hashCode());
+        result = prime * result + (companiesHouseactivity ? 1231 : 1237);
+        result = prime * result + ((sicCode == null) ? 0 : sicCode.hashCode());
+        result = prime * result + ((sicDescription == null) ? 0 : sicDescription.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj){
+            return true;
+        }
+        if (obj == null){
+            return false;
+        }
+        if (getClass() != obj.getClass()){
+            return false;
+        }
+        CombinedSicActivitiesApiModel other = (CombinedSicActivitiesApiModel) obj;
+        if (activityDescription == null) {
+            if (other.activityDescription != null)
+                return false;
+        } else if (!activityDescription.equals(other.activityDescription)) {
+            return false;
+        }
+        if (id == null) {
+            if (other.id != null)
+                return false;
+        } else if (!id.equals(other.id)){
+            return false;
+        }
+        if (companiesHouseactivity != other.companiesHouseactivity){
+            return false;
+        }
+        if (sicCode == null) {
+            if (other.sicCode != null)
```

Comment:
As per Tom's previous comment, it's preferential to surround `if` statements with braces (helps with readability and prevents accidental bugs)

Reply:
It is possible to auto-add braces in Eclipse using Cleanup.
Or IntelliJ using its formatter:
https://stackoverflow.com/questions/59685766/automatically-add-curly-braces-to-single-line-if-for-while-etc-statements-in-int